src/Runtime/Scripts/cshtml5.js

```diff
@@ -325,6 +334,158 @@ document.errorCallback = function (error, IndexOfNextUnmodifiedJSCallInList) {
 	window.onCallBack.OnCallbackFromJavaScriptError(idWhereErrorCallbackArgsAreStored);
 }
 
+document.rerouteMouseEvents = function (id) {
+    document.onmouseup = function(e) {
+        if(e.doNotReroute == undefined)
+        {
+               var element = document.getElementByIdSafe(id);
+               document.reroute(e, element);

```

Comment:
do not use 'getElementByIdSafe' in js scripts
apply this everywhere in this function
```suggestion
               var element = document.getElementById(id);
               if (element) {
                   document.reroute(e, element);
               }
```